pkg/products/monitoring/reconciler.go

```diff
@@ -203,6 +203,11 @@ func (r *Reconciler) Reconcile(ctx context.Context, installation *integreatlyv1a
 		return phase, err
 	}
 
+	isMultiAZCluster, err := resources.IsMultiAZCluster(ctx, serverClient)
```

Comment:
Did you test that this IsMultiAzCluster works then? Or did you check with @sergioifg94 ?

Reply:
I've tested this on both a single and multi-AZ cluster.